src/server/script_mgr.h

```diff
@@ -54,6 +54,9 @@ class ScriptMgr {
   // Returns a list of all scripts in the database with their sha and body.
   std::vector<std::pair<std::string, ScriptData>> GetAll() const;
 
+  // Returns if scripts run as global transactions by default
+  bool AreGlobalByDefault() const;
```

Comment:
oh, i see. we can do it per script basis, though for the MVP it suffices.

Reply:
I wrote above that its a dirty fix. Doing it properly is requires loading ALL the script ahead - it can't be quickly added into our simple function that is present there.

Reply:
Ok, writing it's dirty fix does not provide an explanation of what's not done or what's missing. Worth expanding in the commit description what works now and what's missing.